api-server/src/summarization/SummarizationController.ts

```diff
@@ -21,4 +22,29 @@ export class SummarizationController {
       newEvent.description,
     );
   }
+
+  @Post('get-brief-description')
+  @ApiOperation({
+    summary: 'returns a brief description based on the provided description',
```

Comment:
```suggestion
    summary: 'returns a brief description based on the provided full description',
```

Minor nitpick intended to make it clearer that we're talking about a summary of the longer description.